third_party/chromium/media/filters/chunk_demuxer.cc

```diff
@@ -342,6 +348,8 @@ bool ChunkDemuxerStream::UpdateVideoConfig(const VideoDecoderConfig& config,
     DCHECK_EQ(state_, UNINITIALIZED);
 #if defined(STARBOARD)
     stream_ = std::make_unique<SourceBufferStream>(mime_type_, config, media_log);
+    if (is_video_buffer_budget_override_enabled_) stream_->EnableVideoBufferBudgetOverride();
+    LOG(INFO) << "SourceBufferStream created with video buffer budget of:" << stream_->GetMemoryLimit();
```

Comment:
The value is already logged in SourceBufferStream ctor and SourceBufferStream::EnableVideoBufferBudgetOverride(), we can remove the log here and avoid introducing the new function GetMemoryLimit().

Reply:
now removed.